app/setup.py

```diff
@@ -111,6 +111,8 @@ def create_app(  # noqa: C901  pylint: disable=too-complex, too-many-statements
     application = Flask(__name__, template_folder="../templates")
     application.config.from_object(settings)
 
+    application.jinja_env.trim_blocks = True
+    application.jinja_env.lstrip_blocks = True
```

Comment:
We should consider grouping these with the other jinja environment changes from line 191, possibly in a `setup_jinja_env` method.